### DISCUSSION TITLE
Add new roles in symfony?

### DISCUSSION THREAD

**Discussion by thisism-ali** [symfony/symfony#46399] [Q&A]
My task is to develop an application that helps accountants and their clients communicate with each other. The website is going to have 3 types of users; an admin, an accountant and a client.

I just created my user entity with php bin/console make:user but how do I add "accountant" and "client" user roles?

**Comment by HeahDude** [comment]
Hello @thisism-ali, there are no predefined roles in Symfony.
You cannot really add roles, you just return them from an instance of `UserInterface` using `getRoles()`, independently from how you chose to assign them.
See https://symfony.com/doc/current/security.html#roles.
